style.css

```diff
@@ -61,10 +62,23 @@ h1, h2, h3 {
     font-weight: 400;
 }
 
-html, body, #app {
+html, body, #app, .app-fader {
     height: 100%;
 }
 
+.app-fader {
+    transition: opacity var(--fade-time) ease-in-out;
+    opacity: 0;
+}
+
+.app-fader.exited, .app-fader.exiting {
```

Comment:
Some of these CSS classes are redundant, but since we're taking them directly from the transition state name, I thought it best to explicitly name them here for clarity.